lib/exd/escript/remoter/zmtp.ex

```diff
@@ -6,21 +6,27 @@ defmodule Exd.Escript.Remoter.Zmtp do
 
   @behaviour Exd.Escript.Remoter
 
-  def applications() do
+  def applications(_) do
     init_zmtp
-    :hello_client.start_link({:local, __MODULE__}, 'zmq-tcp://127.0.0.1:10900', {[], [], []})
-    :hello_client.call(__MODULE__, {"options", [], []})
+    {:ok, zmq_address} = :application.get_env(:exd, :addr) |> IO.inspect
+    {:ok, client} = Hello.Client.start_link({:local, __MODULE__}, zmq_address, {[], [], []})
+    {:ok, apps} = Hello.Client.call(__MODULE__, {"options", %{}, []})
+    apps
   end
 
-  def remote(_api, _method, _payload) do
-    :test
+  def remote(api, method, payload) do
+    metadata = %{}
+    metadata = Map.put_new(metadata, :module, api["module"] |> String.to_atom)
+    metadata = Map.put_new(metadata, :payload, payload)
+    {:ok, response} = Hello.Client.call(__MODULE__, {method, metadata, []})
+    response
   end
 
   def init_zmtp() do
     Application.put_env :sasl, :sasl_error_logger, :false, persistent: true
     Application.put_env :lager, :handlers, [], persistent: true
     Application.ensure_all_started(:ezmq)
+    Application.ensure_all_started(:exometer)
```

Comment:
are you sure that we should start `exometer`? may be `exometer_core` or just `hello`?